code/src/components/ForMore.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { ContainerWrapper, FlexWrapper, SectionWrapperGradiant, SectionHeadings } from 'GlobalStyles';
+
+// react component
+
+export const ForMore = () => {
+  return (
+    <SectionWrapperGradiant>
+      <ContainerWrapper>
+        <FlexWrapper>
+          <SectionHeadingsGr>for more</SectionHeadingsGr>
+          <HeaderIconsForMore>
+            <a href="https://www.linkedin.com/in/nina-berggren" target="_blank" rel="noreferrer" aria-label="Go to Linked In page">
+              <i className="fab fa-linkedin-in" alt="LinkedIn icon" />
+            </a>
+            <a href="https://github.com/NinaBerggren" target="_blank" rel="noreferrer" aria-label="Go to Github page">
+              <i className="fab fa-github" alt="GitHub icon" />
+            </a>
+            <a href="https://stackoverflow.com/users/19384658/nina-berggren" target="_blank" rel="noreferrer" aria-label="Go to Stackoverflow page">
+              <i className="fab fa-stack-overflow" alt="Stack Overflow icon" />
+            </a>
+          </HeaderIconsForMore>
+        </FlexWrapper>
+      </ContainerWrapper>
+    </SectionWrapperGradiant>
+  )
+}
+
```

Comment:
I love the gradient you tried to do here. Even if I know you're not completely satisfied with it I think it looks really good.